internal/server/handlers_test.go

```diff
@@ -1241,7 +1241,7 @@ func TestAccountsHandler(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer os.RemoveAll(s.opts.DataDir)
+	// defer os.RemoveAll(s.opts.DataDir)
```

Comment:
Did you mean to leave this commented out?

Reply:
thanks updated and removed the comment